tests/validation/tests/v3_api/test_k3s_airgap.py

```diff
@@ -0,0 +1,216 @@
+import os
+import time
+from lib.aws import AWS_USER
+from .common import AmazonWebServices
+from .test_airgap import (AG_HOST_NAME, BASTION_ID, NUMBER_OF_INSTANCES,
+                          TARBALL_TYPE, AGENT_REG_CMD,
+                          get_bastion_node, prepare_registries_mirror_on_node,
+                          add_cleaned_images, run_command_on_airgap_node,
+                          deploy_bastion_server, wait_for_airgap_pods_ready,
+                          deploy_noauth_bastion_server, copy_certs_to_node,
+                          trust_certs_on_node, add_tarball_to_node)
+
+ARCH = os.environ.get("RANCHER_ARCH", "amd64")
+RANCHER_K3S_VERSION = os.environ.get("RANCHER_K3S_VERSION", "")
+K3S_SERVER_OPTIONS = os.environ.get("RANCHER_K3S_SERVER_OPTIONS", "")
+K3S_AGENT_OPTIONS = os.environ.get("RANCHER_K3S_AGENT_OPTIONS", "")
+
+
+def test_deploy_airgap_k3s_system_default_registry():
```

Comment:
There's lots of re-used code between these tests? Can we parameterize the test with the install method and use common logic between them?